seqr/views/utils/export_utils.py

```diff
@@ -76,10 +79,11 @@ def export_multiple_files(files, zip_filename, file_format='csv', add_header_pre
                 content += '\n'.join([
                     DELIMITERS[file_format].join([row.get(key) or blank_value for key in header]) for row in rows
                 ])
-                if not isinstance(content, unicode):
-                    content = unicode(content, errors='ignore')
-                zip_file.writestr('{}.{}'.format(filename.encode('utf-8'), file_format), content)
+                if isinstance(content, str):
+                    content = content.encode('utf-8')
```

Comment:
why is this necessary? `str` should be able to handle any of the data it would receive

Reply:
The idea behind this is to strip out the unicode chars from the input data.  We could have a variety of ways to do the stripping depending on the input data types:
1. Python 2 `str`: do decoding to an 'ascii' text with either `str.decode()` or `future.builtins.str()`
2. Python 2 `unicode` or `future.builtins.newstr`: We need to encode the data to binary and then decode to `ascii` texts
3. Python 3 `str`: We need to encode the data to binary and then decode to `ascii` texts
So, it does encoding for #2 and #3 cases and does decoding for all cases.

Simple `str()` will cause errors in below cases:
. `str(data, errors='ignore')` if `data` is a Python 3 `str`
. `str(data)` or `str(data, errors='ignore')` if `data is a Python 2 `str` with unicode chars in it
We have to encode Python 2 `str` with unicodes and Python 3 `str` into binary before we can do `str()`.